cylc/flow/cfgspec/globalcfg.py

```diff
@@ -775,6 +775,9 @@ def load(self):
             fname = os.path.join(conf_path_str, self.CONF_BASENAME)
             if os.access(fname, os.F_OK | os.R_OK):
                 self.loadcfg(fname, upgrader.USER_CONFIG)
+            else:
+                LOG.warning(
+                    f'No global.cylc file at CYLC_CONF_PATH: {conf_path_str}')
```

Comment:
Is this useful? I noticed there was a previous commit to raise an error if the path doesn't exist, but that commit was reverted

Reply:
Because this environment variable can get copied to remote platforms (which aren't on the same filesystem) this warning is useful some of the time, and misleading some of the time.

Reply:
Ok, removing it on the basis that being misleading some of the time is worse than not being helpful some of the time